### PR TITLE
Merge backports.csv autobot version update fix

### PR DESCRIPTION
**Pull request by oblute**
Checklist
* [x] Used a fork of the feedstock to propose changes
* [ ] Bumped the build number (if the version is unchanged)
* [x] Reset the build number to `0` (if the version changed)
* [x] [Re-rendered]( https://conda-forge.org/docs/conda_smithy.html#how-to-re-render ) with the latest `conda-smithy` (Use the phrase <code>@<space/>conda-forge-admin, please rerender</code> in a comment in this PR for automated rerendering)
* [x] Ensured the license file is being packaged.

### CODE REVIEW

**Comment by oblute** [comment]
@conda-forge-admin, please rerender

**Comment by oblute** [comment]
@rmax can you please merge this? thanks!

**Comment by oblute** [comment]
@conda-forge/help-python is it possible to add me as a maintainer for this package so I can merge this update? haven't heard back from the original maintainer since pushing this update fix.

**Comment by beckermr** [comment]
Yes, but you need to contact @conda-forge/backports.csv-feedstock. The staged-recipes folks at help-python do not have commit rights here.

**Comment by oblute** [comment]
@conda-forge/core is it possible to add me as a maintainer for this package so I can merge this update? haven't heard back from the original maintainer (who is the only member of @conda-forge/backports-csv) since pushing this update fix.

**Comment by ocefpaf** [comment]
> @conda-forge/core is it possible to add me as a maintainer for this package so I can merge this update? 

You'll be as soon as we merge this. Usually we wait a few days for the current maintainer. So let's get this in. Thanks for the PR.

File: recipe/meta.yaml [hunk 1]
```diff
@@ -1,23 +1,23 @@
...
+{% set sha256sum = "538e04f193710a7fd2522168e65daa1fd7089766b2721dc5621c9b205c7a285e" %}
 
 package:
   name: {{ name }}
   version: {{ version }}
 
 source:
-  fn: {{ name }}-{{ version }}.tar.gz
-  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  url: https://github.com/ryanhiebert/backports.csv/archive/master.zip
```
Comment (ocefpaf) [inline recipe/meta.yaml]: Why do you need to GH source? If we can use the PyPI one it usually helps avoid a number of issues, like the checksum change when the releases is modified upstream.

Comment (oblute) [reply recipe/meta.yaml]: @ocefpaf my team has just been told in the past that the github link can be preferred, so it's just my usual choice of formatting. Happy to change if needed

Comment (CJ-Wright) [reply recipe/meta.yaml]: At the very least we should avoid using the `master.zip` since that is not released.

Comment (oblute) [reply recipe/meta.yaml]: @ocefpaf @CJ-Wright i made the change back to the PyPi link. thanks

Comment (ocefpaf) [reply recipe/meta.yaml]: > @ocefpaf my team has just been told in the past that the github link can be preferred, so it's just my usual choice of formatting. Happy to change if needed

I'm probably guilty of that one. I do (did?) prefer GH releases b/c they usually had tests and licenses that were not present in the PyPI tarball. However, these cases are becoming more rare and the checksum change on GH releases are quite annoying to deal with.

**Pull request merged**